<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="settingLayout">
        <item>
         <widget class="QGroupBox" name="serialSettingGroup">
          <property name="title">
           <string>串口设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
               <item>
                <widget class="QLabel" name="portNumLlabel">
                 <property name="text">
                  <string>端口号</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="portNumBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
               <item>
                <widget class="QLabel" name="baudRateLabel">
                 <property name="text">
                  <string>波特率</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="baudRateBox">
                 <item>
                  <property name="text">
                   <string>1200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4800</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>38400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>57600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>115200</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
               <item>
                <widget class="QLabel" name="dataBitsLabel">
                 <property name="text">
                  <string>数据位</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="dataBitsBox">
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
               <item>
                <widget class="QLabel" name="parityLabel">
                 <property name="text">
                  <string>校验位</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="parityBox">
                 <item>
                  <property name="text">
                   <string>无</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>奇</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>偶</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
               <item>
                <widget class="QLabel" name="stopBitsLabel">
                 <property name="text">
                  <string>停止位</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="stopBitsBox">
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1.5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="openUSARTBtn">
               <property name="text">
                <string>打开串口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="closeUSARTBtn">
               <property name="text">
                <string>关闭串口</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QFrame" name="ledFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="ledLight">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel{
	background-color: gray;
	border-radius: 10px
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="ledLabel">
                <property name="text">
                 <string>状态灯</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="receiveSettingGroup">
          <property name="title">
           <string>接收设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QGroupBox" name="displayMethodGroup">
             <property name="title">
              <string>显示方式</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QCheckBox" name="hexDisplayCheckBox">
                <property name="text">
                 <string>HEX显示</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="displayTimeCheckBox">
                <property name="text">
                 <string>显示时间</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="clearReceiveBtm">
               <property name="text">
                <string>清空接收</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pauseBtn">
               <property name="text">
                <string>暂停接收</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveDataBtn">
               <property name="text">
                <string>保存数据</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="sendSettingGroup">
          <property name="title">
           <string>发送设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QLineEdit" name="sendingCycleLine">
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="sendMethodGroup">
             <property name="title">
              <string>发送方式</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="autoSendCheckBox">
                <property name="text">
                 <string>自动发送</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="hexSendCheckBox">
                <property name="text">
                 <string>HEX发送</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="openFileBtn">
               <property name="text">
                <string>打开文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendFileBtn">
               <property name="text">
                <string>发送文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearSendingBtn">
               <property name="text">
                <string>清除发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sendingCycleLabel">
             <property name="text">
              <string>发送周期（ms）</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="displayLayout" stretch="2,1">
        <item>
         <layout class="QVBoxLayout" name="receiveLayout">
          <item>
           <widget class="QLabel" name="receiveAreaLabel">
            <property name="text">
             <string>接收区</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="receiveTextBrowser"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="sendLayout">
          <item>
           <widget class="QLabel" name="sendAreaLabel">
            <property name="text">
             <string>发送区</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="sendTextEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
